framework/source/class/qx/event/handler/GestureCore.js

```diff
@@ -422,7 +422,7 @@ qx.Bootstrap.define("qx.event.handler.GestureCore", {
         this.stopMomentum(this.__momentum[domEvent.pointerId]);
       }
       // do nothing if we don't need to scroll
-      if ((Math.abs(deltaY) < 1 && Math.abs(deltaX) < 1) || this.__stopMomentum[oldTimeoutId]) {
+      if ((Math.abs(deltaY) < 1 && Math.abs(deltaX) < 1) || this.__stopMomentum[oldTimeoutId] || !this.__window) {
         delete this.__stopMomentum[oldTimeoutId];
```

Comment:
It is not possible, in the `build`  version, to access privates from other classes.